### ISSUE TITLE
Failing distrib tests

### ISSUE THREAD

**Issue by skeating** [sbmlteam/libsbml#58]
TestDistribToAnnotation("cauchy");
  TestDistribToAnnotation("laplace");

TestDistribToAnnotationWithMeans("cauchy");
  TestDistribToAnnotationWithMeans("laplace");

fail

**Comment by skeating** [comment]
@luciansmith Sorry I dont get the logic. The reason these tests are failing is because the function being converted has two arguments but the first allowed argument is 1.
Both Cauchy and Laplace can have one argument  ??

```
bool
DistribToAnnotationConverter::addFunctionDefinitionWith(Model* model, const string& id, ASTNodeType_t type, unsigned int nargs)
{
  DistribASTPlugin dastp;
  vector<unsigned int> nallowed = dastp.getNumAllowedChildren(type);
  if (nallowed.size() == 0) {
    return true;
  }
  if (nargs != nallowed[0]) {
    //Cannot create an annotation version
    return true;
  } 

```

**Comment by luciansmith** [comment]
This should be fixed with https://github.com/sbmlteam/libsbml/pull/61

The logic is that the old annotation version of cauchy/laplace required two arguments for both.  In the new version, at Chris's request, we also allowed 1 argument.  When I made that change, I didn't realize that the converter would break, as it was assuming that the first value in the 'allowed arguments vector' was 'the number of arguments used in the old annotation version'.
